server/db/models/game.js

```diff
@@ -341,15 +341,21 @@ gameSchema.methods.dealCards = function() {
     .then(function(numCards){
         numCardsToDeal = numCards;
         cardsToDeal = deck.slice(0, numCardsToDeal)
-        self.update({deck: deck.slice(numCardsToDeal)});
-        return self.getPlayers()
+        self.deck = deck.slice(numCardsToDeal);
+        return self.save();
+    })
+    .then(function() {
```

Comment:
this function would take the saved game so you could do this instead:
.then(function(game) {
    game.getPlayers()
})